mongoid.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |s|
   # activemodel 7.0.0 cannot be used due to Class#descendants issue
   # See: https://github.com/rails/rails/pull/43951
   s.add_dependency("activemodel", ['>=5.1', '<7.1', '!= 7.0.0'])
+  s.add_dependency("i18n", '>=1.1')
```

Comment:
Note that `i18n` is a dependency of `activemodel` already, this just enforces the min version.

I have confirmed that `i18n` 1.1 works with `activemodel` 5.1.6 and greater (users on AM 5.1.0~5.1.5 will need to upgrade to 5.1.6+, which they should be doing anyway to get security patches)